src/content/docs/tutorial-create-alerts/improve-with-alerts.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: "Manage your alert quality"
+metaDescription: "How manage your alerts and notifications"
+---
+
+import AQMdash from 'images/alerts_screenshot-crop_AQM-dash.webp'
+
+import AQMflow from 'images/alerts_diagram_AQM-flow.webp'
+
+As time goes on, your number of alerts will grow. This can lead to problems for your organization if they're not managed correctly. Additionally, your alerts will give you crucial information you can use to improve your system, and if you don't take advantage of that information, you won't be using your alerts to their fullest potential. Below, we've provided information on managing the quality of your alerts to prevent things like alert fatigue, as well as how you can use alerts to gather data and drive positive impact to your organization.
+
+<img
+    title="AQM dashboard"
+    alt="A screenshot displaying a view of an AQM dashboard in New Relic"
+    src={AQMdash}
+/>
+
+## Manage alert quality
+
```

Comment:
Do we have to brand this as QM?

Reply:
Suggest rebranding to "optimizing alerts"